File: Spring-boot-JWT-with-angular-Part-1-main/src/main/java/com/jwt/secqurity/purchase/entity/Purchase.java
package com.jwt.secqurity.purchase.entity;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import javax.persistence.Entity;

@Entity
@Getter
@Setter
@ToString
public class Purchase extends PurchaseBase{
    private String catagory;
    private String sub_catagory;
    private String brand_name;
    private String product_name;
    private String sku;
    private String  supplier_name;
    private String status;
    private Long qty;
    private String unit;
    private Long purchase_price;
    private Long total;
    private Long paid;
    private String payment_status;
    private String date;

}

File: SalesAndInventory/src/app/purchase/purchase-Service/purchase.ts
export interface Purchase{
   
    purchase_id: number;
    catagory:string;
    sub_catagory:string;
    product_name: string;
    brand_name: string;
   
    sku:string;
    supplier_name: string;
    status: string;
    qty: number;
    unit: string;
    purchase_price: number;
    total: number;
    paid: number;
    payment_status:string;
    date:Date;
    }